invenio_app_rdm/config.py

```diff
@@ -767,3 +783,17 @@ def files_rest_permission_factory(obj, action):
     ("invenio_app_rdm/default_static_page.html", "Default"),
 ]
 """List of available templates for pages."""
+
+FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES = [
+    # 'root://eospublic'
+]
+"""URI prefixes of files their checksums should be verified"""
+
+FILES_INTEGRITY_REPORT_TEMPLATE = "invenio_app_rdm/report_email.html"
```

Comment:
```suggestion
FILES_INTEGRITY_REPORT_TEMPLATE = "invenio_app_rdm/emails/files_integrity_report.html"
```

minor: for convention and easier tracking of all emails